lib/api/v3/work_packages/schema/work_package_schema_representer.rb

```diff
@@ -92,9 +92,6 @@ def initialize(schema, context)
 
           schema :spent_time,
                  type: 'Duration',
-                 show_if: -> (_) do
-                   current_user_allowed_to(:view_time_entries, context: represented.project)
-                 end,
```

Comment:
I've removed this part since I argue we do not have to explicitly hide the schema link. If the user is unable to write it, it will be shown in the schema with form embedded. This will not contain any sensitive information, only that there is a `spentTime` attribute